doc/source/examples/extended_examples/gui/executable.rst

```diff
@@ -0,0 +1,119 @@
+:orphan:
+
+.. _gui_example:
+
+=======================================
+Create a GUI app in Python with PySide6
+=======================================
+
+This example shows how to create a graphical user interface (GUI) app in Python that use PyMAPDL to compute the deflection of a squared shape beam.
```

Comment:
I am reopenning this conversation because vale wants me to use the word "app" instead of "application".
See [here](https://github.com/ansys/pymapdl/runs/14000024791)
Should I still use the word "application" ?

Reply:
You can follow the Vale rules, it was a suggestion :)